src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -107,9 +107,15 @@ class VolumeBar extends Slider {
   updateARIAAttributes(event) {
     // Current value of volume bar as a percentage
     const volume = Math.round((this.player_.volume() * 100)).toString();
```

Comment:
what about changing this `const` to a `let`, and then setting it to `0`, if `muted` is true? That way, we aren't duplicated the setting of the `aria-volumenow` and `aria-valuetext`